google-cloud-firestore/src/main/java/com/google/cloud/firestore/telemetry/TraceUtil.java

```diff
@@ -75,7 +75,7 @@ public interface TraceUtil {
    * @return An instance of the TraceUtil class.
    */
   static TraceUtil getInstance(@Nonnull FirestoreOptions firestoreOptions) {
```

Comment:
since we are updating the class, can we also annotate the util class internalApi? it is marked public to be accessible outside the package, but it is not a public API.

Reply:
Good point. Done.